huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_pool_test.go

```diff
@@ -129,11 +130,11 @@ func TestAccCCENodePool_data_volume_encryption(t *testing.T) {
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck: func() {
-			testAccPreCheck(t)
-			testAccPreCheckKms(t)
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckKms(t)
```

Comment:
it seems that PreCheckKms is unnecessary, we can run the test directly, please update it in your next PR.